android/src/main/jni/RNCProgressBarMeasurementsManager.cpp

```diff
@@ -0,0 +1,75 @@
+#include "RNCProgressBarMeasurementsManager.h"
+
+#include <fbjni/fbjni.h>
+#include <react/renderer/core/conversions.h>
+#include <react/renderer/components/view/conversions.h>
+#include <react/jni/ReadableNativeMap.h>
+
+using namespace facebook::jni;
+
+namespace facebook
+{
+  namespace react
+  {
+
+    Size RNCProgressBarMeasurementsManager::measure(
```

Comment:
Could you write a comment from where did you take analogous code with some links and a short explanation of what is done here if needed?

Reply:
Added in https://github.com/react-native-progress-view/progress-bar-android/pull/90/commits/722b62f6ac3c4e72ba05030d81941adf21857d4d.